source/includes/read/project.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use MongoDB\Client;
+
+$client = new Client('<connection string>');
+
+// start-db-coll
+$db = $client->sample_restaurants;
+$collection = $db->restaurants;
```

Comment:
I have the feeling that the `$db` variable adds nothing to the explanations. 

```suggestion
$collection = $client->sample_restaurants->restaurants;
```

OR 

```suggestion
$collection = $client->selectCollection('sample_restaurants', 'restaurants');
```